cardstack/cardstack
# Create small, medium, large screen widths

### Added
- control for medium width
- variables for medium and large width
- tests
- icon for medium width (same SVG as "responsive" but rotated and clipped)

### Changed
- some refactoring of control methods
- rename responsive to small
- rename full-width to large
- default themer pane width is tied to card size

### How to test
Visit the themer. Click the small-med-large icons at the top of the page and see the size change.

<img width="1404" alt="Screen Shot 2020-01-29 at 5 05 11 PM" src="https://user-images.githubusercontent.com/16627268/73401663-92f2f380-42b9-11ea-8954-ddc0ac2eb887.png">
<img width="1382" alt="Screen Shot 2020-01-29 at 5 05 20 PM" src="https://user-images.githubusercontent.com/16627268/73401664-92f2f380-42b9-11ea-9fb5-ff5472e8c3d1.png">
<img width="1386" alt="Screen Shot 2020-01-29 at 5 05 28 PM" src="https://user-images.githubusercontent.com/16627268/73401665-92f2f380-42b9-11ea-8af0-274268e955a0.png">
